package.json

```diff
@@ -1,16 +1,18 @@
 {
 	"type": "module",
 	"scripts": {
+		"build": "echo 'Please choose a target: build:firefox or build:chrome, or build:production'",
```

Comment:
We could make it so that `build` calls `build:firefox` and `build:chrome`, so that there is one command that builds everything that needs to be built, which would later be the command that is run on CI.

Reply:
Nervermind, there's already a `build:production` script for that purpose.